src/rootPages/Designer/ui_work_interface_list_copyPage.js

```diff
@@ -0,0 +1,354 @@
+                     required: true,
+                     placeholder: `${L("{0} - copy", [oldName])}`,
+                     labelWidth: 110,
+                     on: {
+                        onAfterRender() {
+                           AB.ClassUI.CYPRESS_REF(
+                              this,
+                              "ui_work_interface_list_newPage_blank_name"
+                           );
+                        },
+                     },
+                  },
+                  {
+                     margin: 5,
+                     cols: [
+                        { fillspace: true },
+                        {
+                           view: "button",
+                           id: this.ids.buttonCancel,
+                           value: L("Cancel"),
+                           css: "ab-cancel-button",
+                           autowidth: true,
+                           click: () => {
+                              this.cancel();
+                           },
+                           on: {
+                              onAfterRender() {
+                                 AB.ClassUI.CYPRESS_REF(this);
+                              },
+                           },
+                        },
+                        {
+                           view: "button",
+                           id: this.ids.buttonSave,
+                           css: "webix_primary",
+                           value: L("Paste Page"),
+                           autowidth: true,
+                           type: "form",
+                           click: () => {
+                              return this.save();
+                           },
+                           on: {
+                              onAfterRender() {
+                                 AB.ClassUI.CYPRESS_REF(this);
+                              },
+                           },
+                        },
+                     ],
+                  },
+               ],
+            }
+         };
+      }
+
+      async init(AB, data) {
+         this.AB = AB;
+         this.data = data;
+         console.log(data);
+
+         webix.ui(this.ui(data.name));
+         webix.extend($$(this.ids.component), webix.ProgressBar);
+
+         this.$form = $$(this.ids.form);
+
+         this.$component = $$(this.ids.component);
+
+         // if there was an error saving the values from our form.
+         this.on("save.error", (err) => {
+            this.onError(err);
+         });
+
+         // if the values we provided were successfully saved.
+         this.on("save.successful", () => {
+            this.onSuccess();
+         });
+      }
+
+      /**
+       * @method applicationLoad()
+       * prepare ourself with the current application
+       * @param {ABApplication} application
+       */
+      applicationLoad(application) {
+         super.applicationLoad(application)
+         var options = [{ id: "-", value: L("[Root page]") }];
```

Comment:
We mentioned in our Standup Meeting a couple of days ago about not putting [] around our text.  As it makes it look like we have a placeholder we haven't updated.